Add kinepolis page (#37)
rustnl/rustnl2025: src/pages/kinepolis.md

---
layout: "../layouts/MarkdownLayout.astro"
title: "Conference Venue: Kinepolis"
slug: "kinepolis"
---

> The RustWeek 2025 conference will be held in <a href=" https://kinepolis.nl/bioscopen/kinepolis-jaarbeurs-utrecht/info/">**Kinepolis Jaarbeurs Utrecht**</a>. This venue is a cinema in the center of Utrecht, right next to the Central Station.

Reviewer: ```suggestion
The RustWeek 2025 conference will be held in <a href=" https://kinepolis.nl/bioscopen/kinepolis-jaarbeurs-utrecht/info/">**Kinepolis Jaarbeurs Utrecht**</a>. This venue is a cinema in the center of Utrecht, right next to the Central Station.

Note: please keep in mind that the workshops (Thu May 15) are not held in the conference venue, but in the second venue, called DUS. See [the week schedule](/#schedule).
```

Reviewer: Not sure we really need to add this here, but this is the typical (and understandable) mistake we want people to avoid.

<img style="width: 100%" src="public/images/venues/kinepolis/Kinepolis-Jaarbeurs-Utrecht3-1200x900.jpg" />

## Conference rooms

The talks will be held in cinema rooms, which has a couple of amazing features. First, a massive cinema-grade screen will make it easy to follow the talks from anywhere in the room. Second, the seats are some of the most comfortable you'll ever encounter at a conference.

<img style="width: 100%" src="public/images/venues/kinepolis/Kinepolis-Jaarbeurs-Utrecht-3.jpg" />

## Socializing

> In the same building, we have a dining area called Speys, which is perfect for eating and socializing. There will enough seats, tables and power outlets for everyone to talk, code and hang out. Food and drinks is of course included with your ticket.

Reviewer: ```suggestion
The catering area, called Speys, is a huge space; perfect for eating and socializing, and hosting our sponsor's booths. It allows for 1200 people, which is more than enough for the 800 attendees we hope to welcome. There will seats, tables and power outlets for everyone to talk, code and hang out. Lunch, snacks, and drinks are, of course, included with your ticket.
```

<img style="width: 100%" src="public/images/venues/kinepolis/Speys_01-1920px.jpg">
<img style="width: 100%" src="public/images/venues/kinepolis/Speys_03-1920px.jpg">

## Getting there

The address of the venue is:

> **Kinepolis Jaarbeurs\
> Jaarbeursboulevard 300\
> 3521 BC Utrecht\
> Nederland**

> The most convenient method to get there is by train, since it's next to the train station.

Reviewer: ```suggestion
The most convenient method to get there is by train, since it's next to the train station.
More information on how to get to Utrecht and the conference can be found in the [Where](/#where) section.
```
